rendering_resource_manager_service/deployment/rrm/populateRRM.txt

```diff
@@ -2,10 +2,16 @@
 # of the rendering resource manager.
 
 # Livre
```

Comment:
Ok, I reformatted the whole file. It's only for information anyway so the formatting does not really matter.